python/mp/rosary.py

```diff
@@ -272,60 +379,110 @@ def clear_effects(self):
         # thing wonky?
         self.add_effect('set_color', 'all', 0, 0, 0)
 
+
     @dm.expose()
-    def del_effect(self, id):
-        """Delete an active effect by id."""
+    def clear_effects_fade(self):
+        """
+        Just calling clear_effects() is jarring, let's ease it in
+        """
 
-        effect = self.effect(id)
+        for eff in self.effects:
+            eff.fade_out(30)
```

Comment:
I'd originally played with the idea of having the "clear all running effects" as part of the trigger class (in prefire maybe). Maybe I'll revive that.